src/firebase/event-paths-get-all.js

```diff
@@ -0,0 +1,14 @@
+const initializeHttpClient = require('./http/initialize-http-client')
+const documentPath = require('./builders/document-path')
+const maxEvents = require('./constants/max-events')
+
+const eventsPath = documentPath({ collection: 'events' })
+const eventKeysUrl = `${eventsPath}?pageSize=${maxEvents}&mask.fieldPaths=key`
```

Comment:
@abroadbent Despite what Firebase's REST documentation suggests, there isn't a public way to delete a whole collection of data at once.

Instead, I had to get a list of all of the event keys and then delete each one.

The `pageSize` query parameter used above isn't documented, but my original `get` request was only returning 100 keys. I gleaned the `pageSize` parameter from the beta API docs. 😰 

The `mask` query parameter is mentioned, but it only links to this [http.proto](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto) source file, so I had to read the code comments and work backward from `{ mask: { fieldPaths: [ 'key' ] } }` to arrive at `mask.fieldPaths=key`. Basically, educated guesswork. 😕